nix/onchain.nix

```diff
@@ -178,8 +212,8 @@ in
             pkgs-latest = import liqwid-nix.nixpkgs-latest { inherit system; };
             pkgs = import liqwid-nix.nixpkgs { inherit system; };
 
-            fourmolu = pkgs-latest.haskell.packages.ghc924.fourmolu_0_9_0_0;
-            applyRefact = pkgs.haskell.packages.ghc924.apply-refact_0_10_0_0;
+            fourmolu = projectConfig.fourmolu.package;
+            applyRefact = projectConfig.applyRefact.package;
```

Comment:
Could you do the same for `hlint`, `cabalFmt`, and `hasktags`? It only makes sense to include here.

Reply:
Sure